### ISSUE TITLE
Generational  ZGC not supported

### ISSUE THREAD

**Issue by loyispa** [microsoft/gctoolkit#364]
The ZGC log pattern seems to have changed since jdk21, do you have a plan to support it?

Known changes for **CYCLE_START**:

before:

```
[20.134s][info ][gc          ] GC(1) Garbage Collection (System.gc()) 22M(1%)->24M(1%)
```

after:
```
[251.570s][info][gc          ] GC(5) Major Collection (Proactive) 684M(23%)->346M(12%) 0.473s
```

**Comment by karianna** [comment]
Yes, Pull Requests are also welcome here though (it's an ever moving target!)

**Comment by kcpeppe** [comment]
Hi, I have work on genZ underway. I'm willing to push the work in my local branch if there is interest

**Comment by loyispa** [comment]
> Hi, I have work on genZ underway. I'm willing to push the work in my local branch if there is interest

@kcpeppe  I am sure interested.  But sorry for my naive question, why is it a local branch not the official repository?

**Comment by loyispa** [comment]
I also  have a preliminary implementation regarding genz, and can meet my own needs ,don't know if it should be submited.

**Comment by karianna** [comment]
You should both push branches as part of a PR request and let's have a look at them!

**Comment by kcpeppe** [comment]
Yes, please push and let's see if we can integrate them.

**Comment by AlinaHovakimyan** [comment]
Hi everyone, may I ask when the next version is planned to be released? We are also looking forward for the generational ZGC support and I see that the latest version available in maven central is from a year ago.
